atlasdb-ete-tests/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM adoptopenjdk/openjdk11:x86_64-debian-jre-11.0.18_10
+FROM amd64/eclipse-temurin:11-jdk
```

we may want to just use the same as circle, e.g.

https://github.com/palantir/atlasdb/blob/aa3429ba2543c118efaac19d6eb27b57b2e20c70/.circleci/config.yml#L6

```suggestion
FROM cimg/openjdk:11.0.10-node
```

---

Wasn't aware we use node in the build process (maybe for the docs?). But either of these looks fine to me

---

Our circle templates include node for web builds, though I think for this we could just use cimg/openjdk

---

```suggestion
FROM cimg/openjdk:11.0
```